docs/fe-oss-apis/attention/sdpa_bwd_sm120.md

```
@@ -60,28 +64,32 @@ can be bitwise non-deterministic when a q-tile receives contributions from
 ## Kernel design and optimizations
 
-### The three-kernel chain
+### The kernel chain
 
-One backward call is three launches, overlapped with programmatic dependent
-launch (PDL) so each kernel's prologue runs under its predecessor's tail:
+One backward call is three launches (four under GQA), overlapped with
+programmatic dependent launch (PDL) so each kernel's prologue runs under its
+predecessor's tail:
 
 ```
```

markdownlint-cli2 (0.23.2)

[warning] Fenced code blocks should have a language specified

(MD040, fenced-code-language)